src/localizer.cpp

```diff
@@ -215,9 +215,6 @@ QHash<int, QByteArray> Localizer::roleNames() const {
 }
 
 int Localizer::rowCount(const QModelIndex& index) const {
-  if (!index.isValid()) {
```

Comment:
I think this has broken the localizer unit tests, could you explain the reasoning for the change? Maybe we need to rethink how the localizer is tested.

Reply:
Are you able to see the languages? Because in the latest build the languages, at least on my computer are totally empty if I don't apply this change.

Reply:
Actually, I confirm it, `rowCount()` should return 0 only when used in tables. Any other `rowCount()` impl ignores the model index.

Reply:
You are correct, without this change I am not seeing the languages.